src/Documentation/sidebar.json

```diff
@@ -120,6 +120,80 @@
         "label": "Managing External Data",
```

Comment:
remove this section, docs as part of this PR if you intend to replace. It's fine to do this when we finalize the structure first, but should be removed/replaced as part of this PR.

Reply:
> remove this section

I am removing it.